README.md

```diff
@@ -60,6 +60,23 @@ consumer.metric.reporters=io.strimzi.kafka.metrics.KafkaPrometheusMetricsReporte
 consumer.auto.include.jmx.reporter=false
 ```
 
+When setting configurations for the Prometheus metrics reporter, they also need to be set with the `admin.`, `producer.` and `consumer.`.
+For example, to set the `listener` to `http://:8081`:
+```properties
+metric.reporters=io.strimzi.kafka.metrics.KafkaPrometheusMetricsReporter
+prometheus.metrics.reporter.listener=http://:8081
+auto.include.jmx.reporter=false
+admin.metric.reporters=io.strimzi.kafka.metrics.KafkaPrometheusMetricsReporter
+admin.prometheus.metrics.reporter.listener=http://:8081
+admin.auto.include.jmx.reporter=false
+producer.metric.reporters=io.strimzi.kafka.metrics.KafkaPrometheusMetricsReporter
+producer.prometheus.metrics.reporter.listener=http://:8081
+producer.auto.include.jmx.reporter=false
+consumer.metric.reporters=io.strimzi.kafka.metrics.KafkaPrometheusMetricsReporter
+consumer.prometheus.metrics.reporter.listener=http://:8081
+consumer.auto.include.jmx.reporter=false
```

Comment:
Wider point, I don't find the `admin.` and `producer.` and `consumer.` prefixes particularly intuitive.  If I had to guess what they did, my immediate thought would be that they control an aspect related to the ordinary Kafka Admin, Producer and Consumer clients, not something particular to Streams or Connect.    I'm not really too familiar with either Streams or Connect, so maybe it is obvious to someone working in that space.

Reply:
Streams and Connect can be seen as "regular" applications and for any of their interactions with Kafka they use internal consumer, producer or admin clients. Both Streams and Connect use these prefixes to pass configurations to all the internal clients they use.

Reply:
ok. so it should be familiar to those working with Streams/Connect.